docs/guides/qiskit-function-templates.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Qiskit Function templates
+description: Learn how to use and build Qiskit Function templates.
+---
+
+
+# Introduction to Qiskit Function templates
+
+[Qiskit Function templates](https://github.com/qiskit-community/qiskit-function-templates) are an open-source collection of realistic code examples that give you access to some of the latest research workflows that leverage tools like the Qiskit SDK and [Qiskit addons](https://quantum.cloud.ibm.com/docs/en/guides/addons). These pre-built workflows take learnings from practical experiments and package them in a way that you can customize and reuse in your own research so that you don't have to build from scratch. All templates are readily deployable to [Qiskit Serverless](./serverless) where you can easily leverage classical compute as part of running your workflow.
+
+Templates are available in two varieties:
+
+- *Template implementations*: Specialized code examples that implement advanced techniques from real-world experiments in areas like chemistry and physics.
+- *Base templates*: References for developing your own original Qiskit Function that highlight best-practices in interface development, code formatting, unit testing, and more.
+
+## Template implementations
+
+Qiskit Function template implementations are organized by application area. Currently included in the collection is a physics template for Hamiltonian simulation using the [AQC-Tensor Qiskit addon](./qiskit-addons-aqc) and a chemistry template for electronic structure with the implicit solvent model using the [SQD Qiskit addon](./qiskit-addons-sqd).
+Resources to get started with these two templates are available at the links below:
+
+- Electronic structure simulation with implicit solvent model: [template source files](https://github.com/qiskit-community/qiskit-function-templates/tree/main/chemistry/sqd_pcm) and [guide](./function-template-chemistry-workflow)
+- Hamiltonian simulation: [template source files](https://github.com/qiskit-community/qiskit-function-templates/tree/main/physics/hamiltonian_simulation) and [guide](./function-template-hamiltonian-simulation)
+
+
+## Base templates
+
+The *base templates* are references for developing your own original Qiskit Function. They are designed to help you kickstart the development process by illustrating best practices in interface development, code formatting, I/O handling, error messages, checkpoints, unit testing, and more. The repository itself is exposed as a [GitHub template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) so that the CI/CD implementation can be easily extended.
```

Comment:
```suggestion
The *base templates* are references for developing your own original Qiskit Function. They are designed to help you kickstart the development process by illustrating best practices in interface development, code formatting, I/O handling, error messages, checkpoints, unit testing, and more. The repository is exposed as a [GitHub template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) so that the CI/CD implementation can be easily extended.
```